clients/py/sapphirepy/tests/test_e2e.py

```diff
@@ -49,24 +52,28 @@ class TestEndToEnd(unittest.TestCase):
     def setUp(self):
         account: LocalAccount = Account.from_key("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80")  # pylint: disable=no-value-for-parameter
 
+
         w3 = Web3(Web3.HTTPProvider('http://localhost:8545'))
+        # w3 = Web3(Web3.HTTPProvider('https://testnet.sapphire.oasis.io'))
         w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
-        self.w3 = w3 = sapphire.wrap(w3)
+        self.w3_no_signer = Web3(Web3.HTTPProvider('http://localhost:8545'))
```

Comment:
Can we test Websockets too?

Reply:
Ok, will add Websocket tests in a separate PR. For now, the latest changes are in https://github.com/oasisprotocol/sapphire-paratime/pull/396. I think we can close this PR.